packages/snap/.eslintrc.js

```diff
@@ -9,6 +9,18 @@ module.exports = {
         'import/no-nodejs-modules': ['error', { allow: ['buffer', 'crypto'] }],
       },
     },
+    {
+      files: ['**/*.test.ts', '**/*.test.js'],
+      extends: ['@metamask/eslint-config-jest'],
+      rules: {
+        '@typescript-eslint/no-shadow': [
+          'error',
+          { allow: ['describe', 'expect', 'it'] },
+        ],
+        '@typescript-eslint/unbound-method': 'off',
+        'no-console': 'off',
```

Comment:
I think we should remove the no console.

Reply:
addressed here [444afde](https://github.com/MetaMask/snap-watch-only/pull/2/commits/444afde80235c2d9f9425acdb6c24c08515ecc0a)